spring-cloud-dataflow-server/docker-compose-influxdb.yml

```diff
@@ -35,6 +35,8 @@ services:
   dataflow-server:
     image: springcloud/spring-cloud-dataflow-server:${DATAFLOW_VERSION:?DATAFLOW_VERSION is not set!}
     container_name: dataflow-server
+    volumes:
+      - '/tmp/apps:/apps/'
```

Comment:
Any specific reasons why we need this volume mount? I don't see it being used in the associated changes in this PR, though.

Reply:
Nice catch! I will remove it. 
I used this volume to register and test the locally build metrics enabled Task apps.